package.json

```diff
@@ -22,49 +22,41 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.2.0",
-    "ember-cli": "1.13.15",
+    "broccoli-concat": "^2.2.0",
+    "broccoli-funnel": "^1.0.1",
```

Comment:
many of these dependencies actually seem like prod dependencies, otherwise those installing will not be able to successfully use this addon.